### PR TITLE
Relocate eth_protocolVersion and web3_clientVersion

### PR DESCRIPTION
**Pull request by njgheorghita**
### What was wrong?
- Moved `eth_protocolVersion` endpoint from `web3.version.ethereum` to `web3.eth.protocolVersion`
- Moved `web3_clientVersion` endpoint from `web3.version.node` to `web3.clientVersion`

Upon approval of this pr, I'll create a pr to raise deprecation warnings in v4

#### Cute Animal Picture
image

### CODE REVIEW

**Comment by njgheorghita** [comment]
Now that `Version.api` is the only method left in the `Version` module, any thoughts on fully deprecating the `Version` module and moving `api` to the `web3` namespace (`web3.api`)

File: tests/core/eth-module/test_eth_protocolVersion.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def test_eth_protocolVersion(web3):
+    assert web3.eth.protocolVersion == '99'
```
Comment (kclowes) [inline tests/core/eth-module/test_eth_protocolVersion.py]: Why is this 99?

Comment (pipermerriam) [reply tests/core/eth-module/test_eth_protocolVersion.py]: I believe it's talking to `eth-tester` and should probably be updated internally to be `63`

Comment (njgheorghita) [reply tests/core/eth-module/test_eth_protocolVersion.py]: Hmmm, strange - according to [this line](https://github.com/ethereum/web3.py/blob/master/web3/providers/eth_tester/defaults.py#L165) it should be returning `63`. Will dive deeper into this tomorrow

File: docs/web3.version.rst [hunk 2]
```diff
@@ -27,23 +27,18 @@ The following properties are available on the ``web3.eth`` namespace.
...
+    Method is deprecated in favor of ``web3.clientVersion``.
+
     * Delegates to ``web3_clientVersion`` RPC Method
 
     Returns the current client version.
 
     .. code-block:: python
 
-        >>> web3.version.node
+        >>> web3.clientVersion
```
Comment (kclowes) [inline docs/web3.version.rst]: I know this is merged already, but I noticed that there isn't documentation for `web3.clientVersion`. Do you mind adding it when you get a chance? :pray:

**Pull request merged**